gadgets/Makefile

```diff
@@ -89,6 +90,17 @@ $(GADGETS_README):
 		ln -sf ../../gadgets/$@ $(ROOT_DIR)../docs/gadgets/$(shell dirname $@).mdx ; \
 	fi
 
+.PHONY: $(GADGETS_README_DEV)
+$(GADGETS_README_DEV):
```

Comment:
If I run `make -C gadgets audit_seccomp/dev.md`, it changes the `audit_seccomp/dev.md`. Same for other gadgets. Please update them. 

BTW, we should verify these files are update in the CI. I suggest adding a new job at the same level of `gadgets-unittest`. Actually, the job will be very similar, but after running the `image inspect`, we will need to verify that no files has been modified (like in the `documentation-checks` job):

```
git diff --exit-code HEAD --
```

To do that, you will need to create a Makefile target like (use `build` as example):

```
gadgets-readme-dev: $(GADGETS_README_DEV)
```